internal/types/array_test.go

```diff
@@ -77,3 +77,78 @@ func TestArray(t *testing.T) {
 		assert.Equal(t, int32(42), b.s[0])
 	})
 }
+
+func TestArrayContains(t *testing.T) {
+	for name, tc := range map[string]struct {
+		array    *Array
+		filter   any
+		expected bool
+	}{
+		"string": {
```

Comment:
WOuld you please use CamelCase for test keys for the code similarity? Thank you.

Reply:
So, this https://github.com/FerretDB/github-actions/blob/7ace04553a01a45fb914cb3928bbcde55a4abe21/detect-matching-pr/main_test.go#L19 format is wrong then?

Reply:
Yeah, in this file it's even not `camelCase` but rather `TitleCase` :)